.github/workflows/test-pull-request.yml

```diff
@@ -8,7 +8,6 @@ on:
   workflow_dispatch:
 
 jobs:
-
   swift-code-checks:
     name: Code Tests
     runs-on: macos-latest
```

Comment:
FYI, you should rebase to pick up this change from @dcaunt: https://github.com/IABTechLab/uid2-ios-sdk/commit/a8b3dbca95a54ee447655ec2bb08dc79a8d3f45a